### PR TITLE
Real-robot place integration

### PR DESCRIPTION
**Pull request by cpaxton**
## Motivation and Context

- Place experiments on the real robot
- Testing hierarchical policy
- Making sure grasp planner code is clean and easier to use

## Changelog

- Fixed bugs in observation handling
- Update obs handler to make sure its getting goal object
- Correctly search for object on start receptacle in the real world not just any object

## Types of changes

- [X] Docs change / refactoring / dependency upgrade
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [X] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [X] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: src/home_robot/home_robot/motion/stretch.py [hunk 1]
```diff
@@ -61,12 +65,13 @@
         0.01,  # arm
         0.0,  # gripper rpy
         0.0,  # wrist roll
-        0.0,  # wrist pitch
+        -0.7,  # wrist pitch
```
Comment (yvsriram) [inline src/home_robot/home_robot/motion/stretch.py:68]: Will we be doing this for both the navigations? Will remember to do this in sim. May protrude scene in rare cases but gripper down with object held may not be practical.

**Review by yvsriram — APPROVED** [review]
LGTM if tested. Thank you!

**Comment by cpaxton** [comment]
Yeah I think we should with both. It needs to pack the gripper away to move to destination without hitting anything.

Best,
Chris

Sent from my phone
________________________________

File: src/home_robot/home_robot/motion/stretch.py (hunk 1 above)
Comment (cpaxton) [reply src/home_robot/home_robot/motion/stretch.py:68]: I think we need to do this, yes; otherwise it will hit stuff when it moves right?

**Pull request merged**